### ISSUE TITLE
Give more information when `import_bibtex` fails to parse an entry?

### ISSUE THREAD

**Issue by ali-ramadhan** [Azzaare/Bibliography.jl#5]
I'm parsing a large-ish BibTeX file (~1,000 lines, ~80 entries) and it seems that when `import_bibtex` fails to parse an entry it prints an error

```julia
julia> import_bibtex("oceananigans.bib")
ERROR: Missing the volume field(s).
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] BibInternal.BibTeX.Article(::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::Dict{String,String}) at /home/alir/.julia/packages/BibInternal/mZf28/src/bibtex.jl:102
 [3] BibInternal.BibTeX.Article(::String, ::Dict{String,String}) at /home/alir/.julia/packages/BibInternal/mZf28/src/bibtex.jl:137
 [4] make_bibtex_entry(::String, ::String, ::Dict{String,String}) at /home/alir/.julia/packages/BibInternal/mZf28/src/bibtex.jl:14
 [5] macro expansion at /home/alir/.julia/packages/Automa/fyalL/src/codegen.jl:137 [inlined]
 [6] parse(::String) at /home/alir/.julia/packages/BibParser/Qhq5E/src/bibtex.jl:137
 [7] parse_file at /home/alir/.julia/packages/BibParser/Qhq5E/src/bibtex.jl:144 [inlined]
 [8] import_bibtex(::String) at /home/alir/.julia/packages/Bibliography/ywTv6/src/bibtex.jl:2
 [9] top-level scope at REPL[2]:1
```

Is it possible for `import_bibtex` to also print maybe the specific entry or a line number in the `.bib` file so it is easy to find and fix the bad entry?

**Comment by Azzaare** [comment]
The line number would be harder, but it is pretty to print the id/key, for instance:
```julia
ERROR: LoadError: Entry CitekeyArticle is missing the volume field(s).
```

If this is enough, it will be included in the next release (v0.1.3)

**Comment by ali-ramadhan** [comment]
Awesome, thank you! Yeah it's a minor issue but would make debugging much faster.
